src/components/tutorials-sidebar/types.ts

```diff
@@ -39,8 +41,13 @@ export interface ListItemProps {
 	text: string
 }
 
+export interface TutorialListItemProps extends ListItemProps {
+	tutorialId: uuid
+	collectionId: uuid
+}
+
 export interface SectionListProps {
-	items: ListItemProps[]
+	children: ReactNode
```

Comment:
Have update this prop to be `children` for flexibility:
- tutorial landing and collection views pass regular `ListItem` for collection sidebar items
- tutorial view passes new `TutorialListItem` for tutorial sidebar items, and these hook into the progress state

Reply:
💭 should this be:
```suggestion
	children: ReactNode[]
```

Reply:
☝️ I'm not sure! I see `ReactNode` in a ton of other places in our code base... but have never dug super deep into the types here, both seem valid 🤷‍♂️ For consistency I'd lean towards keeping it as-is, but happy to do a deep dive or adjust if there are any strong opinions.

Reply:
Yeah this one is kinda weird! It feels like `ReactNode[]` would be the way to go because the `map` is sending multiple children to this component, but TypeScript also isn't complaining about `ReactNode` being used so maybe it's fine? 🤔

Reply:
Here's a relatively definitive answer: in `@types/react`, the `PropsWithChildren` type (used for `React.FC`, for example) uses `ReactNode`:

```
type PropsWithChildren<P> = P & { children?: ReactNode | undefined };
```

ref: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/fb66211010980641f01bcfe8ffbc0cc3d71989ca/types/react/index.d.ts#L799